Source/SIMPLib/DataContainers/DataContainerArrayProxy.cpp

```diff
@@ -544,3 +544,13 @@ QMap<QString, DataContainerProxy> DataContainerArrayProxy::readMap(QJsonArray js
   }
   return map;
 }
+
+QMap<QString, DataContainerProxy> DataContainerArrayProxy::getDataContainers()
+{
+	return dataContainers;
```

Comment:
getDataContainers and setDataContainers still require indentation checks.  The comment blocks above their definitions are also missing.